src/main.zig

```diff
@@ -142,6 +145,53 @@ fn help(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.All
     try std.io.getStdOut().writer().print("\n", .{});
 }
 
+fn init() Command_Error!void {
```

Comment:
`init()` should take the same parameters as the rest of the commands, so `name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.Allocator`. Zig will be mad at you because you're not using them, but that's fine, you can just add a `_ = arg;` in the function body for each arg to silence the warning.